src/pages/HomePage.js

```diff
@@ -0,0 +1,78 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { fetchData } from '../redux/data/dataSlice';
+
+const HomePage = () => {
+  const dispatch = useDispatch();
+  const countriesData = useSelector((store) => store.data);
+  const defaultContinents = [
+    {
```

Comment:
- [ ] Dear @Reggeditt, good job making the changes. But as commented out by the previous reviewer We need to follow these [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), including:
`Colors (select one main color for your website)`.
 `Typography: font face, size, and weight`.
 `Layout: composition and space between elements`. And currently your design misses
* color composition, Your buttons are black and this does not move well with your design. Please try to stick to the primary color you have, (pick something similar to your primary/pink color) 
* Spacing/Size, Your design overflows on a side on the detail page, Please take a look into that. 
**Remember as a frontend engineer we are expected to deliver something identical to the original design.**
### Your home page 
![image](https://user-images.githubusercontent.com/69077061/236645464-a77c22a3-4f9e-4e5a-9a22-26f0054ee391.png)

### Your detail page on 375px screen.
![image](https://user-images.githubusercontent.com/69077061/236645493-c9874b02-d137-4bb2-a184-24f446dbea27.png)

Reply:
Yes please.
thanks for pointing it out

Reply:
- [ ] Again here as requested by the previous reviewer, we need to follow the given design guidelines. For example the give font is `lato` and `gill sans`, whereas you have used the `Roboto` font. Kindly fix this
![image](https://user-images.githubusercontent.com/81932478/236649964-4b26ebd0-440c-4081-9406-33c5082b3a4a.png)

- [ ] As requested by the previous reviewer again, your design overflows, hiding part of the text, and figures in the app and this is still at the recommended 375px for mobile view
![image](https://user-images.githubusercontent.com/81932478/236650169-3f6b2ea3-fb9e-4ddd-a8b6-4b48157b3a26.png)


Your design
![image](https://user-images.githubusercontent.com/81932478/236650107-e25b9f7e-f415-4cad-9482-7b3397e63b3a.png)

Given design
![Uploading image.png…]()